specs/app_settings/01_main.rst

```diff
@@ -0,0 +1,236 @@
+==========================================================
+App settings
+==========================================================
+
+=====  ============  =========  ==========
+epic      title       author     created
+=====  ============  =========  ==========
+`82`_  App settings  tschubotz  2019-01-18
+=====  ============  =========  ==========
+
+.. _82: https://github.com/gnosis/safe/issues/82
+
+.. _Main:
+
+
+#. `Main`_
+#. `User Interface`_
+#. `Technical Details`_
+
+.. sectnum::
+.. contents:: Table of Contents
+    :local:
+    :depth: 2
+
+Problem Definition
+---------------------
+
+- There are some app features that we need to provide the user access to.
+    - Access Terms of Use
+    - Access Proviacy Policy
+    - Change password
+    - Enable/disable biometric authentication
+    - Option to Give feedback (Send us an email)
+    - Option to rate the app in the store.
+    - View app version (For bug reporting)
+    - View used open source licenses
+
+
+Inputs
+-----------
+
+ - Depending on the use case: Password and biometrics information, Terms, Privacy Policy
+
+
+Outputs
+----------
+
+Outputs depend on the selected option.
+- Open webpage (Terms & Privacy Policy)
+- Start change password flow
+- Biometric auth has been enabled / disabled
+- Open up email program (Give feedback)
+- Open store for rating / writing review (Rate app)
+- App version is displayed
+- View used open source licenses
+
+
+Use Case Scenarios
+-----------------------
+
+App version
+~~~~~~~~~~~~~~~~
+
+Preconditions
++++++++++++++
+
+- App is set up with a password
+
+Steps
++++++
+
+1. Open app settings
+
+Postconditions
+++++++++++++++
+
+- User can see app version in way so we know which version they are using.
+
+
+View Terms of Use
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Inherits from `App version`_
+
+Steps
++++++
+
+2. Open Terms of Use
+3. Browser opens with https://safe.gnosis.io/terms
```

Comment:
I'd say this is the post condition.

Reply:
> _Launch web browser and load https://safe.gnosis.io/terms_

Against that since I try to write from the user's perspective, so would use passive here.

> I'd say this is the post condition.

Thought about that too, make more sense, will change.